iothub_client/src/iothub_client_core_ll.c

```diff
@@ -39,6 +39,8 @@
 
 #define LOG_ERROR_RESULT LogError("result = %s", MU_ENUM_TO_STRING(IOTHUB_CLIENT_RESULT, result));
 #define INDEFINITE_TIME ((time_t)(-1))
+#define ERROR_CODE_BECAUSE_DESTROY 0
```

Comment:
Nit: somehow integrate twin into this since it’s being used twin specific?
`TWIN_ERROR_CODE_BECAUSE_DESTROY`

Reply:
We should keep this generic because it might be used in other cases
ERROR_CODE_BECAUSE_CLIENT_DESTROY?

Reply:
Makes sense. Shall do it.